test/build-with-timeout/build-with-low-timeout.yaml

```diff
@@ -18,7 +18,7 @@ metadata:
   labels:
     expect: failed
 spec:
-  timeout: 50s
+  timeout: "50s"
```

Comment:
These quotes aren't necessary right? I don't feel strongly about removing them, I just want to make sure this all works if a user doesn't explicitly quote the string.

Reply:
Quotes are not necessary. When I changed the type into string I might have gone on a spree to add quotes everywhere :D I will follow up with a PR to address all your comments.